tindergames/src/main/java/org/teamcifo/logic/BoardGameManager.java

```diff
@@ -0,0 +1,31 @@
+package org.teamcifo.logic;
+
+import org.teamcifo.utils.Helpers;
+import org.teamcifo.domain.BoardGame;
+
+public class BoardGameManager {
+    // we need how to decide how to storage boardgames
+
+    public static void createBoardGame(){
+        // will create a singe boardgame
+    }
+
+    public static void createMultipleBoardGames(int number){
+        // can create multiple boardgames at the same time
+        //maybe only for testing?
+    }
```

Comment:
If it is a method for testing purposes, it's better to move it onto a test utils class

Reply:
Method keeped in this class until other test classes require it